src/agent/state/global.rs

```diff
@@ -279,23 +275,6 @@ where
                 .await
                 .price_accounts
                 .insert(*account_key, *account);
-
-            // Notify the Pythd API adapter that this account has changed.
-            // As the account data might differ between the two networks
-            // we only notify the adapter of the primary network updates.
-            if let Network::Primary = network {
-                StateApi::global_store_update(
-                    state,
-                    Identifier::new(account_key.to_bytes()),
-                    account.agg.price,
-                    account.agg.conf,
-                    account.agg.status,
-                    account.valid_slot,
-                    account.agg.pub_slot,
-                )
-                .await
-                .map_err(|_| anyhow!("failed to notify pythd adapter of account update"))?;
-            }
```

Comment:
Why is this one removed?

Reply:
It's moved into the Prices service, otherwise you have Prices -> GlobalStore -> Prices, I left a comment in the commit but just to expand further. The only places this is called is from oracle. I renamed the existing `update_price` in `Prices` to `update_local_price`, and as the logic for updating is already there, it made more sense to also add `update_global_price` as well, and then moving this logic into Pricing, given the function is already defined there.

I'm pretty confident this is a better design and how it really should have been, especially given the existence of `update_price`.